core/controllers/suggestion.py

```diff
@@ -465,6 +466,29 @@ def _get_target_id_to_skill_opportunity_dict(suggestions):
     return opportunity_id_to_opportunity_dict
 
 
+def _construct_exploration_suggestions(suggestions):
+    """Returns exploration suggestions with current exploration content.
+
+    Args:
+        suggestions: list(BaseSuggestion). A list of suggestions.
+
+    Returns:
+        list(dict). List of suggestion dicts with an additional
+        exploration_content_html field representing the target
+        exploration's current content.
+    """
+    suggestion_dicts = []
+    for suggestion in suggestions:
+        exploration = exp_fetchers.get_exploration_by_id(
```

Comment:
@sagangwee Looks like we are making N datastore calls here, this is leading to slow translation-review tab page loading!

Reply:
Note: this was fixed by @vojtechjelinek in https://github.com/oppia/oppia/pull/14164/files#diff-5a316725c3d13e54962250ee01ea2f180103f7c197c949161ddf635e20d4e346R473